moveit_core/collision_detection/include/moveit/collision_detection/collision_detector_allocator.h

```diff
@@ -101,7 +103,10 @@ class CollisionDetectorAllocatorTemplate : public CollisionDetectorAllocator
     return CollisionRobotPtr(new CollisionRobotType(dynamic_cast<const CollisionRobotType&>(*orig)));
   }
 
-  /** Create an allocator for FCL collision detectors */
+  /** Create an allocator for FCL collision detectors.
+   *
+   *  The function is used in the planning scene to allocate the new detector with the corresponding CollisionWorld /
```

Comment:
It's not particularly helpful to describe _where_ a function is used. This would constrain future (different) usages.
Please undo.